include/boost/geometry/algorithms/detail/overlay/get_turn_info.hpp

```diff
@@ -195,10 +209,21 @@ struct touch_interior : public base_turn_handler
         }
         else if (side_qi_p == 1 && side_qk_p == 1 && side_qk_q == -1)
         {
+            if (side_pj_q2 == -1)
+            {
             // Q turns right on the left side of P (test "ML3")
-            // Union: take both operation
+                // Union: take both operations
```

Comment:
tab?

Reply:
I upgraded my VM and apparently the settings are not right yet, will fix. Thanks.

Reply:
Was not a tab but the rest was missing an indent. Fixed now (will push later).